### DISCUSSION TITLE
v1.1.5 - better validations for SD3.5M and Lycoris users

### DISCUSSION THREAD

**Discussion by bghira** [bghira/SimpleTuner#1166] [Announcements]
## Features
- Flow-matching models like SD3 and Flux can use uniform schedule sampling again, mirroring the v0.9.x release cycle from early August
- More model card details for Hugging Face Hub
- SD3.5 Medium: skip-layer guidance for validation outputs to more closely match usual workflow results
- SD3.x: Allow configuring T5 and CLIP padding values (default to empty string)
- Added `--vae_enable_tiling` for reducing VAE overhead on 2048px training for SD3.5 Medium on smaller GPUs
- CLIP score tracking for validations by adding `--evaluation_type=clip` to your config
- LyCORIS training can now have a specific strength set during validations using `--validation_lycoris_strength` to mirror the typical workflows found in ComfyUI etc. A recommended value is 1.0 (default) or 1.3. Using a value lower than 1.0 can help to avoid seeing a model "blow up" when you intend on using it at a lower weight later, anyway.

## Bugfixes
- Torch compile for validation fixed, now works (it did nothing before)
- Torch compile disabled for LyCORIS models
- Better SD3 quantisation performance via quanto by excluding layers from the quantisation
- Flux: default shift value to 3 instead of 1
- SD1.5 LoRA save fixed
- Quanto typo for FP8 fixed
- Multi-caption parquet backend crashing fixed
- Concurrent text embed writes on multi-GPU system file locking issue fixed

## Pull requests
* experimental: remove some layers from quanto by @bghira in https://github.com/bghira/SimpleTuner/pull/1085
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1086
* flux: modify the quanto default excluded layers to be different from sd3 by @bghira in https://github.com/bghira/SimpleTuner/pull/1087
* sd3: allow configuring clip and t5 uncond values by @bghira in https://github.com/bghira/SimpleTuner/pull/1088
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1089
* fix SD3 text embed creation; downgrade to pytorch 2.4.1 by @bghira in https://github.com/bghira/SimpleTuner/pull/1093
* update docs and sd3 parameter defaults by @bghira in https://github.com/bghira/SimpleTuner/pull/1094
* Small link update in TUTORIAL by @rootonchair in https://github.com/bghira/SimpleTuner/pull/1095
* (#1097) resolve sd15 lora save error by @bghira in https://github.com/bghira/SimpleTuner/pull/1102
* fix(typo): correct arg name in warning by @Jannchie in https://github.com/bghira/SimpleTuner/pull/1099
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1103
* Add deduplication of captions by @mhirki in https://github.com/bghira/SimpleTuner/pull/1104
* Throw an error if both --flux_schedule_auto_shift and --flux_schedule_shift are enabled. by @mhirki in https://github.com/bghira/SimpleTuner/pull/1106
* Fix unit test failure after PR #1106 by @mhirki in https://github.com/bghira/SimpleTuner/pull/1107
* Fix gO variable name by @samedii in https://github.com/bghira/SimpleTuner/pull/1108
* disable caption deduplication as it prevents multigpu caching; add warning for sd3 using wrong VAE; cleanly terminate and restart batch text embed writing thread by @bghira in https://github.com/bghira/SimpleTuner/pull/1111
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1112
* sd3: revert enforcement of sd35 flow_matching_loss values by @bghira in https://github.com/bghira/SimpleTuner/pull/1115
* Updating Flux Quickstart Doc with Pre-Trained Model Info by @riffmaster-2001 in https://github.com/bghira/SimpleTuner/pull/1116
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1123
* Fix missing docker dependencies by @Putzzmunta in https://github.com/bghira/SimpleTuner/pull/1126
* Fix multi-caption parquets crashing in multiple locations (Closes #1092) by @AmericanPresidentJimmyCarter in https://github.com/bghira/SimpleTuner/pull/1109
* sd3: add skip layer guidance by @bghira in https://github.com/bghira/SimpleTuner/pull/1125
* sd3: model card detail expansion by @bghira in https://github.com/bghira/SimpleTuner/pull/1130
* flux and sd3 could use uniform sampling instead of beta or sigmoid by @bghira in https://github.com/bghira/SimpleTuner/pull/1129
* Fix random validation errors for good (and restore torch.compile for the validation pipeline at the same time) by @mhirki in https://github.com/bghira/SimpleTuner/pull/1131
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1132
* revamp model card to work by default and provide quanto hints by @bghira in https://github.com/bghira/SimpleTuner/pull/1133
* validation: disable compile for lycoris by @bghira in https://github.com/bghira/SimpleTuner/pull/1136
* add --vae_enable_tiling to encode large res images with less vram used by @bghira in https://github.com/bghira/SimpleTuner/pull/1141
* s3: when file does not exist, handle generic 404 error for headobject by @bghira in https://github.com/bghira/SimpleTuner/pull/1142
* trainer: enable vae tiling when enabled by @bghira in https://github.com/bghira/SimpleTuner/pull/1143
* validation: fix error when torch compile is disabled for lycoris by @bghira in https://github.com/bghira/SimpleTuner/pull/1144
* add clip score tracking by @bghira in https://github.com/bghira/SimpleTuner/pull/1146
* add documentation updates by @bghira in https://github.com/bghira/SimpleTuner/pull/1150
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1151
* metadata: add more ddpm related schedule info to the model card by @bghira in https://github.com/bghira/SimpleTuner/pull/1152
* local data backend should have file locking for writes and reads by @bghira in https://github.com/bghira/SimpleTuner/pull/1160
* chore: ignore rmtree errors by @bghira in https://github.com/bghira/SimpleTuner/pull/1162
* validation: allow setting a non-default strength for validation with lycoris by @bghira in https://github.com/bghira/SimpleTuner/pull/1161
* add more info to model card, refine contents by @bghira in https://github.com/bghira/SimpleTuner/pull/1163
* error out when cache dir path is not found by @bghira in https://github.com/bghira/SimpleTuner/pull/1164
* merge by @bghira in https://github.com/bghira/SimpleTuner/pull/1165

## New Contributors
* @rootonchair made their first contribution in https://github.com/bghira/SimpleTuner/pull/1095
* @Jannchie made their first contribution in https://github.com/bghira/SimpleTuner/pull/1099
* @samedii made their first contribution in https://github.com/bghira/SimpleTuner/pull/1108
* @Putzzmunta made their first contribution in https://github.com/bghira/SimpleTuner/pull/1126

**Full Changelog**: https://github.com/bghira/SimpleTuner/compare/v1.1.4...v1.1.5

<hr /><em>This discussion was created from the release <a href='https://github.com/bghira/SimpleTuner/releases/tag/v1.1.5'>v1.1.5 - better validations for SD3.5M and Lycoris users</a>.</em>
